web/src/helpers/scales.js

```diff
@@ -50,12 +50,15 @@ const solarColor = d3.scaleLinear()
 
 // ** CO2
 const maxCo2 = 800;
```

Comment:
is this still used?

Reply:
still used, but could be omitted / integrated in themes. need to clean up scales and themes in general